app/export_converters.go

```diff
@@ -132,9 +132,9 @@ func ImportReplyFromPost(post *model.ReplyForExport) *ReplyImportData {
 	}
 }
 
-func ImportReactionFromPost(reaction *model.Reaction) *ReactionImportData {
+func ImportReactionFromPost(user *model.User, reaction *model.Reaction) *ReactionImportData {
 	return &ReactionImportData{
-		User:      &reaction.UserId,
+		User:      &user.Username,
```

Comment:
Did this come about because of the naming of this `User` string attribute? Is it unclear from the name whether it's meant to be an id or a username?

Reply:
Maybe, but semantically it is the user who made the reaction in the export/import feature (And it is bound using the username always for all import/export feature)